.github/workflows/ci.yml

```diff
@@ -51,12 +51,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        dist: [bionic, jammy, noble]
```

Comment:
note: we'll need to do some puppet shenanigans / send an email before we merge this